src/main/java/org/ow2/proactive/catalog/util/parser/WorkflowParser.java

```diff
@@ -106,9 +119,6 @@
 
     }
 
-    public WorkflowParser() {
-    }
-
     public List<KeyValueLabelMetadataEntity> parse(InputStream inputStream) throws XMLStreamException {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 30 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)